src/components/CardLists/index.js

```diff
@@ -0,0 +1,35 @@
+import {exercises} from '../../../backend/db';
+import useStore from '../../hooks/useStore';
+import StyledCalendarButton from '../Buttons/CalendarButton/styled';
+import StyledCard from '../Card/styled';
+import StyledHeadline from '../CardHeadline/styled';
+
+import StyledCardlist from './styled';
+
+function CardlistIndex() {
+	const addCard = useStore(state => state.addCard);
+
+	return (
+		<>
+			<p>Alle Übungen</p>
+			<StyledCardlist>
+				{exercises.map(exercise => {
+					return (
+						<StyledCard key={exercise._id}>
```

Comment:
I don't see why you changed this into a button element. Should it rather be a list item since you have a list as parent element?

Reply:
The Cardlist ist now an ul and the cards li Elements. But how do i get rid of the left margin I now have? :(